controllers/localization_controller.go

```diff
@@ -233,8 +228,15 @@ func (r *LocalizationReconciler) reconcile(ctx context.Context, obj *v1alpha1.Lo
 	}
 
 	// create snapshot
-	snapshotName := fmt.Sprintf("%s/snapshots/%s:%s", r.OCIRegistryAddr, obj.Spec.SnapshotTemplate.Name, obj.Spec.SnapshotTemplate.Tag)
-	snapshotDigest, err := r.writeSnapshot(ctx, snapshotName, artifactPath.Name())
+	snapshotName := fmt.Sprintf(
```

Comment:
We definitely need a better way to manage creating the snapshot name, I can see it getting messy.